homeassistant/components/decora_ble/config_flow.py

```diff
@@ -0,0 +1,150 @@
+"""Config flow for Decora BLE."""
+from __future__ import annotations
+
+import logging
+from typing import Any, Optional
+
+from decora_bleak import DECORA_SERVICE_UUID, DecoraBLEDevice
+import voluptuous as vol
+
+from homeassistant.components import bluetooth
+from homeassistant.components.bluetooth import (
+    BluetoothServiceInfoBleak,
+    async_discovered_service_info,
+)
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_ADDRESS, CONF_API_KEY, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+_LOGGER.setLevel(logging.DEBUG)
```

Comment:
Please remove this.  Debug logging should be controlled by the logger integration